packages/react-components/source/react/library/sidebar/SidebarFooter.js

```diff
@@ -31,6 +34,7 @@ const defaultProps = {
   minimized: false,
   profileIcon: null,
   enableSignout: false,
+  tooltip: null,
```

Comment:
Some PDS components go ahead and specify default text. Since this is only for the sign out button, what do you think about adding a default like "Sign out"? I might also rename this something like `signoutTooltip` so no one is confused if it applies to the whole Sidebar.Footer.